### ISSUE TITLE
Missing Sound Design Credits

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#163]
There are no sound designers credited for Ohm's Law.
Names fell through all the comments in #54 and https://github.com/phetsims/QA/issues/313#issuecomment-488394000

![Screen Shot 2020-10-09 at 1 08 55 PM](https://user-images.githubusercontent.com/1902116/95603928-866e8f80-0a31-11eb-9c21-0c9327a8d797.png)

Ashton Morris and possibly Mike Winters need to be added as Sound Design.

@emily-phet or @Ashton-Morris, can you verify if Mike Winters was on this simulation?

**Comment by terracoda** [comment]
Please re-assign to me @terracoda when you verify Michael Winters.

**Comment by terracoda** [comment]
Same as in Resistance in a Wire, we are missing some people including me.
I'll see if there is an issue open.

**Comment by emily-phet** [comment]
@terracoda 

I think only Jesse Greenberg, and Mike Winters are missing. You're on there, @terracoda .

**Comment by terracoda** [comment]
Oh, so I am :-)

**Comment by zepumph** [comment]
Updated and published: 

https://phet.colorado.edu/sims/html/ohms-law/latest/ohms-law_en.html
